src/UI/Art/SpriteArtist/MapTileDetailsArtist.java

```diff
@@ -68,11 +68,11 @@ public static void drawTileDetails(Graphics g, GameMap map, XYCoord tileToDetail
     // Draw the tile coordinates.
     if(SpriteOptions.getCoordinatesEnabled())
     {
-      String coordStr = String.format("(%d, %d)", tileToDetail.xCoord, tileToDetail.yCoord);
+      String coordStr = String.format("(%d,%d)", tileToDetail.xCoord, tileToDetail.yCoord);
```

Comment:
Really trying to make the most of screen real estate?

Reply:
The alignment was bugging me.

Reply:
Hah, pretty sure I tried it both ways and kept the space in there because it looked weird without it.